lib/src/chatlist/chat_list_bloc.dart

```diff
@@ -131,13 +131,27 @@ class ChatListBloc extends Bloc<ChatListEvent, ChatListState> {
       messageListObservable.listen((state) async {
         if (state is MessagesStateSuccess) {
           var uniqueInviteMap = LinkedHashMap<int, int>();
+          Context context = Context();
+          List<int> contacts = await context.getContacts(2, null);
           await Future.forEach(state.messageIds, (messageId) async {
             ChatMsg message = _messageListRepository.get(messageId);
             var contactId = await message.getFromId();
             if (!uniqueInviteMap.containsKey(contactId)) {
               uniqueInviteMap.putIfAbsent(contactId, () => messageId);
             }
           });
+          List<int> removedContacts = List<int>();
+          await Future.forEach(uniqueInviteMap.keys, (contactId) async{
+            if(contacts.contains(contactId)){
+              int newChatId = await context.createChatByMessageId(uniqueInviteMap[contactId]);
+              _messageListRepository.clear();
```

Comment:
This would remove all invite messages, right? So if I have two different invites open, all would be deleted. I think we should only remove all messages which are in the new chat (dc_get_chat_msgs for the newChatId). This should be easier and more reliable than trying to get all relevant messages via the contact.